.npmignore

```diff
@@ -15,3 +15,7 @@ system_test/
 scripts/
 ci/
 .github/
+.eslintrc.js
+.pre-commit-config.yaml
+Jenkinsfile
```

Comment:
Double check after removing Jenkinfile, still BPTP pipeline is good.

Reply:
it't not repoving Jenkinsfile, only exclusion from npm package